prawcore/exceptions.py

```diff
@@ -106,7 +106,7 @@ class Redirect(ResponseException):
     """
 
     def __init__(self, response):
-        """Initialize a Redirect exception instance..
```

Comment:
I also removed this extra period. I hope that's okay.

Reply:
Absolutely. Thanks!